capstone-backend/src/test/java/com/google/sps/data/BookTest.java

```diff
@@ -1,14 +1,11 @@
-package com.google.sps;
+package com.google.sps.data;
 
```

Comment:
I also changed the file structure for the test Java classes ```sps/data``` instead of just ```sps```. I think this is more consistent to because our normal Java classes are also in ```sps/data```

Reply:
Good change! I also found out that this allows methods to be package-private and run them in their sister-packages (both under com.google.sps but in different parent packages)